addons/udes_priorities/models/stock_picking.py

```diff
@@ -81,13 +94,23 @@ def get_priorities(self):
         PriorityGroup = self.env["udes_priorities.priority_group"]
         groups = []
 
-        for group in PriorityGroup.search(self._priority_group_domain()):
-            groups.append(
-                {
-                    "name": group.name,
-                    "priorities": [{"id": p.reference, "name": p.name} for p in group.priority_ids],
-                }
-            )
+        group_domain_kwargs = {}
+        if picking_type_id is None:
+            group_domain_kwargs["picking_type_ids"] = [picking_type_id]
```

Comment:
`picking_type_id` is None here, so why don't just []?

Reply:
should probably be not none but mick wasn't sure why he had put this

Reply:
i will update this in the hht story so we can test it as that is what this function is intended for, as you say it doesn't block this